### PR TITLE
Changed "From" name on Sendgrid emails

### PR DESCRIPTION
**Pull request by hongyi-chen**
## Description

Just changed the "From" name to `Raising the Roof / Chez Toit`

## Testing

Try doing something that requires an email to be sent (verifying email, forgot password, etc.)

### CODE REVIEW

File: src/resolvers/auth.js [hunk 1]
```diff
@@ -16,7 +16,10 @@ const {
 const createResetPasswordEmail = resetToken => {
   const resetPasswordUrl = `${CLIENT_URL}/resetPassword/${resetToken}`;
   return {
-    from: EMAIL_FROM_ADDRESS,
+    from: {
```
Comment (hongyi-chen) [inline src/resolvers/auth.js]: hmm might be worthwhile to extract this entire `from` as a const?

Comment (jzxhuang) [reply src/resolvers/auth.js]: yeah, i think so

**Review by jzxhuang — APPROVED** [review]
extract as constant and you're good

File: src/resolvers/auth.js (hunk 1 above)
Comment (hongyi-chen) [reply src/resolvers/auth.js]: Fixed in a024aec

**Review by jzxhuang — APPROVED** [review]

**Pull request merged**